### PR TITLE
Add panacus histgrowth module

### PR DESCRIPTION
**Pull request by heringerp**
Adds a module for the panacus histgrowth command. This is needed to add panacus statistics to the pangenome pipeline.

## PR checklist

Closes #XXX 

- [x] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
- [x] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [x] Remove all TODO statements.
- [x] Emit the `versions.yml` file.
- [x] Follow the naming conventions.
- [x] Follow the parameters requirements.
- [x] Follow the input/output options guidelines.
- [x] Add a resource `label`
- [x] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [ ] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [x] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [x] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

File: modules/nf-core/panacus/histgrowth/meta.yml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+      description: |
+        Groovy Map containing sample information
+        e.g. `[ id:'test', single_end:false ]`
+  - versions:
+      type: file
+      description: File containing software versions
+      pattern: "versions.yml"
+  - tsv:
+      type: file
+      description: TSV file containing the statistics
```
Comment (subwaystation) [inline modules/nf-core/panacus/histgrowth/meta.yml:53]: Could you please mention here that either a TSV or an HTML will be the output?

File: tests/modules/nf-core/panacus/histgrowth/main.nf [hunk 2]
```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { PANACUS_HISTGROWTH } from '../../../../../modules/nf-core/panacus/histgrowth/main.nf'
+
+workflow test_panacus_histgrowth {
```
Comment (subwaystation) [inline tests/modules/nf-core/panacus/histgrowth/main.nf:7]: My wild guess here is that this is not in accordance with the new nf-test rules, but we will find out once you make this a real PR.

File: tests/modules/nf-core/panacus/histgrowth/nextflow.config [hunk 3]
```diff
@@ -0,0 +1,5 @@
+process {
+
+    publishDir = { "${params.outdir}/${task.process.tokenize(':')[-1].tokenize('_')[0].toLowerCase()}" }
+    
+}
```
Comment (subwaystation) [inline tests/modules/nf-core/panacus/histgrowth/nextflow.config:5]: Missing EOL.

**Review by subwaystation — CHANGES_REQUESTED** [review]
This looks really good already! Please fix the EOL and the explanations. Then make it a real PR @heringerp

**Comment by subwaystation** [comment]
@mashehu The trick didn't work......

**Comment by subwaystation** [comment]
@heringerp It seems the md5sums of Singularity and Docker differ. A little bit strange. Can you test locally?
If you get the desired result regardless if the md5sums are fitting, then revert the test back to just check if the expected files exist. Thanks!

File: tests/config/pytest_modules.yml [hunk 4]
```diff
@@ -2503,6 +2503,14 @@ pairtools/stats:
...
   - tests/modules/nf-core/pairtools/stats/**
 
+panacus/histgrowth:
+  - modules/nf-core/panacus/histgrowth/**
+  - tests/modules/nf-core/panacus/histgrowth/**
+
+panacus/visualize:
+  - modules/nf-core/panacus/visualize/**
+  - tests/modules/nf-core/panacus/visualize/**
+
```
Comment (mashehu) [inline tests/config/pytest_modules.yml:2513]:
```suggestion
```
This was just needed for pytest, your tests are with nf-tests and will be picked up based on their tags.

File: tests/modules/nf-core/panacus/visualize/test.yml
Comment (mashehu) [inline tests/modules/nf-core/panacus/visualize/test.yml:1]: this file is not needed for nf-test

File: tests/modules/nf-core/panacus/histgrowth/test.yml
Comment (mashehu) [inline tests/modules/nf-core/panacus/histgrowth/test.yml:1]: this file is not needed for nf-test

**Review by mashehu — CHANGES_REQUESTED** [review]
still has some relics from pytests 🙂

**Comment by subwaystation** [comment]
Cool, looks good to me now :)

**Review by subwaystation — APPROVED** [review]

**Pull request merged**